pkgs/os-specific/linux/nvidia-x11/generic.nix

```diff
@@ -23,35 +23,42 @@
 with stdenv.lib;
 
 assert (!libsOnly) -> kernel != null;
+assert (versionOlder version "391") -> sha256_32bit != null;
+assert !(versionOlder version "391") -> stdenv.hostPlatform.system == "x86_64-linux";
 
 let
   nameSuffix = optionalString (!libsOnly) "-${kernel.version}";
   pkgSuffix = optionalString (versionOlder version "304") "-pkg0";
+  i686bundled = (versionAtLeast version "391");
```

Comment:
Unnecessary parens

Reply:
I disagree: `i686bundled` is used [here](https://github.com/NixOS/nixpkgs/blob/31b056e3d73728663d68b1e1414be1b950bba4bd/pkgs/os-specific/linux/nvidia-x11/builder.sh#L34). Please explain if I am wrong.

Reply:
I just mean to remove the parentheses, like this:
```nix
i686bundled = versionAtLeast version "391";
```